sktime/forecasting/model_evaluation/tests/test_evaluate.py

```diff
@@ -358,6 +365,38 @@ def test_evaluate_probabilistic(n_columns, scoring):
         pass
 
 
+@pytest.mark.skipif(
+    not run_test_for_class([evaluate] + INTERVAL_METRICS_WITH_PARAMS),
+    reason="run test only if softdeps are present and incrementally (if requested)",
+)
+@pytest.mark.parametrize("n_columns", [1, 2])
+@pytest.mark.parametrize("scoring", INTERVAL_METRICS_WITH_PARAMS)
+def test_evaluate_probabilistic_with_params(n_columns, scoring):
+    """Check that evaluate works with interval, quantile, and distribution forecasts."""
+    y = _make_series(n_columns=n_columns)
+
+    forecaster = NaiveForecaster()
+    cv = SlidingWindowSplitter()
+    try:
```

Comment:
A very general recommendation: try/except should always be avoided in a case where one could to if/else instead, with a known condition.

I suspect in this case, we do know whne the `NotImplementedError` occurs? Could oyu kindly explain? Can we change this to if/else?

Reply:
Thanks. It is indeed redundant. I copy pasted the skeleton of the test from the test above **test_evaluate_probabilistic** . I will remove it. Does not affect the test results.